spec/chewy/index/actions_spec.rb

```diff
@@ -55,7 +55,7 @@
     context do
       before { DummiesIndex.create }
       specify do
-        expect { DummiesIndex.create! }.to raise_error(Elasticsearch::Transport::Transport::Errors::BadRequest).with_message(/index_already_exists_exception.*dummies/)
+        expect { DummiesIndex.create! }.to raise_error(Elasticsearch::Transport::Transport::Errors::BadRequest).with_message(/already exists.*dummies/)
```

Comment:
```suggestion
        expect { DummiesIndex.create! }.to raise_error(Elasticsearch::Transport::Transport::Errors::BadRequest).with_message(/resource_already_exists_exception.*dummies/)
```
https://github.com/elastic/elasticsearch/blob/feab123ba400b150f3dcd04dd27cf57474b70d5a/rest-api-spec/src/main/resources/rest-api-spec/test/indices.rollover/10_basic.yml#L131

Reply:
Discard this. Otherwise, it will stop working on ES5.

Reply:
Just to keep track of the differences:

```
ES 5 [400] {"error":{"root_cause":[{"type":"index_already_exists_exception","reason":"index [dummies/xadaxSc9QhONQCws2qcZzw] already exists","index_uuid":"xadaxSc9QhONQCws2qcZzw","index":"dummies"}],"type":"index_already_exists_exception","reason":"index [dummies/xadaxSc9QhONQCws2qcZzw] already exists","index_uuid":"xadaxSc9QhONQCws2qcZzw","index":"dummies"},"status":400}
ES 6 [400] {"error":{"root_cause":[{"type":"resource_already_exists_exception","reason":"index [dummies/nmx0XrJlQi6so8rl_m4dnw] already exists","index_uuid":"nmx0XrJlQi6so8rl_m4dnw","index":"dummies"}],"type":"resource_already_exists_exception","reason":"index [dummies/nmx0XrJlQi6so8rl_m4dnw] already exists","index_uuid":"nmx0XrJlQi6so8rl_m4dnw","index":"dummies"},"status":400}
 ```